feature-libs/checkout/base/styles/components/steps/review/_review.scss

```diff
@@ -0,0 +1,123 @@
+%cx-checkout-review-shipping {
```

Comment:
After we finish the refactor work, we need UI devs to review these scss changes.